sentry-apache-http-client-5/src/test/kotlin/io/sentry/transport/apache/ApacheHttpClientTransportFactoryTest.kt

```diff
@@ -11,13 +13,16 @@ class ApacheHttpClientTransportFactoryTest {
     @Test
     fun `creates ApacheHttpClientTransport`() {
         val factory = ApacheHttpClientTransportFactory()
-        val options = SentryOptions().apply {
-            setLogger(mock())
-            setSerializer(mock())
-        }
-        val requestDetails = mock<RequestDetails>()
-
-        val transport = factory.create(options, requestDetails)
+        val transport = factory.create(SentryOptions(), mock())
         assertNotNull(transport)
     }
+
+    @Test
+    fun `options timeouts are used when creating ApacheHttpClientTransport`() {
```

Comment:
This test is very questionable. There's nothing we can assert here on the client itself. To write a proper test we would need to set up a mock http server with ability to set connect and response timeouts. Not sure if worth the effort.

Reply:
agreed, can we actually read these values from `CloseableHttpAsyncClient` and see if they are the same as from options? it'd be more useful I guess

Reply:
Only with reflection. There are pros & cons but after all, with a hope that httpclient won't change it internal structure much, i think it's the better way. Code updated.